Cargo.toml

```diff
@@ -27,10 +27,10 @@ serde = "1.0"
 serde_json = "1.0"
 sha2 = "0.9"
 simplelog = "0.8"
-symbolic-common = "7.5"
-symbolic-debuginfo = "7.5"
-symbolic-demangle = "7.5"
-symbolic-minidump = "7.5"
+symbolic-common = "8"
+symbolic-debuginfo = "8"
+symbolic-demangle = "8"
+symbolic-minidump = "8"
```

Comment:
You could consider to require `symbolic` with `features = ["demangle", "minidump"]`. You'll then have a single import via `symbolic::*` and just a single dependency to maintain.

Reply:
Yeah, let's try that. @Eijebong can you give it a spin?

Reply:
Will try it yeah